app/assets/stylesheets/helpers/_contents-list.scss

```diff
@@ -0,0 +1,91 @@
+.contents-list {
+  margin-bottom: $gutter-two-thirds;
+
+  // Always render the contents list above a
+  // back to contents link
+  position: relative;
+  z-index: 1;
+  background: $white;
+  box-shadow: 0 20px 15px -10px $white;
+
+  li {
+    padding-top: $gutter / 4;
+    line-height: 1.3;
```

Comment:
Genuine question - should there be a unit of measurement for the line height or will all browsers safely default to em? (which is what I presume it's supposed to be?)